aikau/src/main/resources/alfresco/header/SearchBox.js

```diff
@@ -592,6 +687,14 @@ define(["dojo/_base/declare",
             this._searchMenu.placeAt(this._searchMenuNode);
             this._searchMenu.startup();
          }
+         else
+         {
+            // When not including the advanced search menu, we'll just include an element that indicates
+            // that indicates the control is still related to search. It's purely for decoration though.
+            domConstruct.create("span", {
+               "class": "alfresco-header-SearchBox-menu--hideAdvancedSearch"
```

Comment:
Really, this is a case of a CSS class describing what it does, not what it is. It should be something like `--advancedSearchDisabled` and then leave the CSS to decide whether to hide it, grey it out, make it smaller, etc.